TombEngine/Scripting/Internal/TEN/Misc/Miscellanous.cpp

```diff
@@ -122,6 +124,45 @@ namespace Misc
 	{
 		return TO_DEGREES(GetCurrentFOV());
 	}
+
+	///Shows the mode of the game camera.
+	//@function GetCameraType
+	//@treturn Misc.CameraType value used by the Main Camera.
+	//@usage
+	//LevelFuncs.OnControlPhase = function() 
+	//	PrintCameraType()
+	//end
+	//
+	//function PrintCameraType()
```

Comment:
I have not, is also one of my doubts. As I told squid, I tried to make a small self working example, but I don't know if it will look ugly in the doc page.

I though in let it there to see how it gets compiled and if it gets too ugly, then always can change for a different example (maybe one that do not touch all the enumerator cases).

Reply:
I did updated it with a shorter version.

```lua
LevelFuncs.OnControlPhase = function() 
  if(Misc.GetCameraType() == CameraType.Combat) then
    --Do your Actions here.
  end
end
```

Reply:
Ok :)